lib/linguist/generated.rb

```diff
@@ -244,7 +244,11 @@ def generated_parser?
     #
     # Returns true or false.
     def generated_postscript?
-      return false unless ['.ps', '.eps'].include? extname
+      return false unless ['.ps', '.eps', '.pfa'].include? extname
+
+      # Type 1 and Type 42 fonts converted to PostScript are stored as hex-encoded byte streams; these
+      # streams are always preceded the `eexec` operator (if Type 1), or the `/sfnts` key (if Type 42).
+      return true if data =~ /(\n|\r\n|\r)\s*(?:currentfile eexec\s+|\/sfnts\s+\[\1<)\h{8,}\1/
```

Comment:
Not really. It's a multiline match for Type 42; and no, `/sfnts` alone would match literally any PostScript program which used that name for an identifier. It's part of a "Dictionary" in PostScript terminology. It'd be like identifying generated JavaScript based solely on an object literal containing `{ sfnts: …`